src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1744,4 +1744,29 @@ String getApiTailUrl(String tail) {
     public GHIssueEvent getIssueEvent(long id) throws IOException {
         return root.retrieve().to(getApiTailUrl("issues/events/" + id), GHIssueEvent.class).wrapUp(root);
     }
+
+    // Only used within getTopics().
```

Comment:
Is this used?

Reply:
Sorry, I mean "listTopics()". I've updated the comment now.